gamemode/modules/workarounds/sh_workarounds.lua

```diff
@@ -87,89 +87,93 @@ timer.Simple(3, function()
-        end
-    end
-
-    local aid = plyMeta.AccountID
-
-    function plyMeta:AccountID(...)
-        return aid(self, ...) or 0
-    end
-end
-
--- Clientside part
-if CLIENT then
-    if DarkRP.disabledDefaults["workarounds"]["Cam function descriptive errors"] then return end
-    local cams3D, cams2D = 0, 0
-    local cam_Start = cam.Start
-
-    function cam.Start(tbl, ...)
-        -- https://github.com/Facepunch/garrysmod-issues/issues/3361
-        if (not istable(tbl)) then
-           argError(tbl, 1, "table")
-        end
+    if not DarkRP.disabledDefaults["workarounds"]["nil SteamID64 and AccountID local server fix"] then return
+        local plyMeta = FindMetaTable("Player")
 
-        if (tbl.type == "3D") then
-            cams3D = cams3D + 1
-        elseif (tbl.type == "2D") then
-            cams2D = cams2D + 1
-        else
-            error("bad argument #1 to '%s' (bad key 'type' - 2D or 3D expected, got %s)", debug.getinfo(1, "n").name, tbl.type, 2)
-        end
-
-        -- Could pcall this but it'd be impossible to
-        -- tell if a render instance was created or not.
-        -- Assume creation/deletion
-        return cam_Start(tbl, ...)
-    end
-
-    local cam_End3D = cam.End3D
+        if SERVER then
+            local sid64 = plyMeta.SteamID64
 
-    function cam.End3D(...)
-        if (cams3D == 0) then
-            error("tried to end invalid render instance", 2)
+            function plyMeta:SteamID64(...)
+                return sid64(self, ...) or "0"
+            end
         end
 
-        cams3D = cams3D - 1
-
-        return cam_End3D(...)
-    end
+        local aid = plyMeta.AccountID
 
-    local cam_End2D = cam.End2D
-
-    function cam.End2D(...)
-        if (cams2D == 0) then
-            error("tried to end invalid render instance", 2)
+        function plyMeta:AccountID(...)
+            return aid(self, ...) or 0
         end
-
-        cams2D = cams2D - 1
-
-        return cam_End2D(...)
     end
+end
 
-    local cams3D2D = 0
-    local cam_Start3D2D = cam.Start3D2D
-
-    function cam.Start3D2D(...)
-        cams3D2D = cams3D2D + 1
-
-        return cam_Start3D2D(...)
-    end
 
-    return
+if CLIENT and not DarkRP.disabledDefaults["workarounds"]["Cam function descriptive errors"] then
+	local cams3D, cams2D = 0, 0
+	local cam_Start = cam.Start
+	cam.Start=function(tbl, ...)
+		-- https://github.com/Facepunch/garrysmod-issues/issues/3361
+		if (not istable(tbl)) then
+		   argError(tbl, 1, "table")
+		end
+		if tbl.type=="3D" then
+			cams3D=cams3D+1
+		elseif tbl.type == "2D" then
+			cams2D=cams2D+1
+		else
+			error("bad argument #1 to '%s' (bad key 'type' - 2D or 3D expected, got %s)",debug.getinfo(1,"n").name,tbl.type, 2)
+		end
+		-- Could pcall this but it'd be impossible to
+		-- tell if a render instance was created or not.
+		-- Assume creation/deletion
+		return cam_Start(tbl,...)
+	end
+	local cam_Start3D,cam_End3D = cam.Start3D,cam.End3D
+	cam.Start2D=function(...)
```

Comment:
I think I agree with @Kefta on the overrides of other functions not being necessary. They fall back to `cam.Start`, so you don't need to override them all.